datalad/log.py

```diff
@@ -259,16 +259,6 @@ def _clear_all(self):
             pb.clear()
 
 
-class NoProgressLog(logging.Filter):
-    def filter(self, record):
-        return not hasattr(record, 'dlm_progress')
-
-
-class OnlyProgressLog(logging.Filter):
-    def filter(self, record):
-        return hasattr(record, 'dlm_progress')
```

Comment:
I wish we declared them protected. Even though unlikely used outside - who knows, and I would have just left them in until 0.15 with deprecation warning emitted

Reply:
I've dropped that commit.